docs/setup/asb/systemd/start-asb.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Run the automated swap backend
+After=start-monero-wallet-rpc.service
+
+[Service]
+WorkingDirectory=/var/external/asb/xmr-btc-swap
+ExecStart=/home/asb/.cargo/bin/cargo run --release --bin asb -- --config /var/external/asb/run/asb-config/config.toml start
```

Comment:
Why do `cargo run` and not just run the built binary?

Reply:
I just did it like this because that way we don't have to have an explicit initial build. It does not really make a difference because Cargo will only build it new if there are changes.